analyzer/src/main/kotlin/managers/Gradle.kt

```diff
@@ -140,7 +140,9 @@ class Gradle(
 
             log.info {
                 "Will apply the following system properties defined in file '$gradlePropertiesFile':\n" +
-                        gradleSystemProperties.joinToString(separator = "\n") { "${it.first} = ${it.second}" }
+                        gradleSystemProperties.joinToString(separator = "\n    ", prefix = "    ") {
```

Comment:
How about then also moving the above "\n" to the `prefix`, so `prefix` and `separator` are the same string?

Reply:
Done. Thought about moving it to a const, but it seemed a bit over-engineered to me, or would you prefer that?

Reply:
No, I absolutely would *not* prefer that, as I agree it to be over-engineered.

Reply:
Approve it then :)

Reply:
But if you wanted to, you could use `"\n\t"`, i.e. use a tab character instead of spaces, to shorten the string a bit. `\t` is also what we use in the [Analyzer](https://github.com/heremaps/oss-review-toolkit/blob/c7dd2d685935bfd7cc97fa942bbb96452b7c6ea8/cli/src/main/kotlin/commands/AnalyzerCommand.kt#L117-L121) for indentation.

Reply:
Done.